paper-slider.html

```diff
@@ -90,6 +90,10 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
I think that given how settings uses Polymer 1.0, `:host-context` is fine.

Since we support Safari 9, add `-webkit-transform: scaleX(-1)` as well (don't worry about any other vendor prefixes).